plugins/hls-module-name-plugin/src/Ide/Plugin/ModuleName.hs

```diff
@@ -4,8 +4,7 @@
 {-# LANGUAGE RecordWildCards   #-}
 {-# LANGUAGE ViewPatterns      #-}
 
-{-# OPTIONS_GHC -Wall -Wwarn -fno-warn-type-defaults #-}
-{-# OPTIONS_GHC -Wno-name-shadowing #-}
+{-# OPTIONS_GHC -Wwarn -fno-warn-type-defaults #-}
```

Comment:
Removing per-file warnings flags that are enabled for all plugins universally via `import warnings` in haskell-language-server.cabal